### PR TITLE
chore: update close stale issues workflow to reflect org standards and update action version to make action run again

### PR DESCRIPTION
**Pull request by peterwoodworth**
We close issues after 7 days without a response, and run the workflow once every 4 hours in our other repos. Update the workflow to reflect this

Updates version of the action used to make action run again, and also limit permissions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/close-stale-issues.yml [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: This issue has not received any attention in 1 year. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
         stale-issue-message: This issue has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
         stale-pr-message: This PR has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
-
+        
+        # These labels are required
         stale-issue-label: closing-soon
-        exempt-issue-labels: no-autoclose,feature-request
```
Comment (RanVaknin) [inline .github/workflows/close-stale-issues.yml:19]: Why did we remove `feature-request` from exemption list?

Comment (peterwoodworth) [reply .github/workflows/close-stale-issues.yml:19]: Because I copy + pasted this from CDK lol

**Comment by RanVaknin** [comment]
@peterwoodworth I would just add a more sensible commit message, or release note describing what you changed :)

**Review by RanVaknin — APPROVED** [review]
Looks Bueno 🔥

**Review by aBurmeseDev — APPROVED** [review]
LGTM 🚀

**Pull request closed without merging**